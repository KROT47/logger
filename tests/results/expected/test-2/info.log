[2018-12-25T13:16:22.197Z] INFO on 222 | { x: 1, a: [Circular] }
[2018-12-25T13:16:22.198Z] INFO on 222 | { a: { a: { a: { a: { a: { a: [Object] } } } } } }
[2018-12-25T13:16:22.198Z] INFO on 222 | { a: { a: { a: { a: { a: { a: [Array] } } } } } }
[2018-12-25T13:16:22.198Z] INFO on 222 | { a: { a: { a: { a: { a: { a: [Function: a] } } } } } }
[2018-12-25T13:16:22.199Z] INFO on 222 | { a: { a: { a: { a: { a: { a: NaN } } } } } }
[2018-12-25T13:16:22.199Z] INFO on 222 | { a: { a: { a: { a: { a: { a: undefined } } } } } }
[2018-12-25T13:16:22.199Z] INFO on 222 | { a: { a: { a: { a: { a: { a: null } } } } } }
[2018-12-25T13:16:22.199Z] INFO on 222 | { a: { a: { a: { a: { a: { a: 1 } } } } } }
[2018-12-25T13:16:22.199Z] INFO on 222 | { a: { a: { a: { a: { a: { a: '1' } } } } } }
[2018-12-25T13:16:22.199Z] INFO on 222 | { a: { a: { a: { a: { a: { a: 2018-12-25T13:16:22.199Z } } } } } }
[2018-12-25T13:16:22.202Z] INFO on 222 | {
[2018-12-25T13:16:22.202Z] INFO on 222 |   a: 
[2018-12-25T13:16:22.202Z] INFO on 222 |    { a: 
[2018-12-25T13:16:22.202Z] INFO on 222 |       { a: 
[2018-12-25T13:16:22.202Z] INFO on 222 |          { a: 
[2018-12-25T13:16:22.202Z] INFO on 222 |             { a: 
[2018-12-25T13:16:22.202Z] INFO on 222 |                { a: Error
[2018-12-25T13:16:22.202Z] INFO on 222 |     at startTest (/home/krot47/repos/logger/tests/tests/test-2.js:46:48)
[2018-12-25T13:16:22.202Z] INFO on 222 |     at Object.<anonymous> (/home/krot47/repos/logger/tests/test.js:47:5)
[2018-12-25T13:16:22.202Z] INFO on 222 |     at Module._compile (module.js:635:30)
[2018-12-25T13:16:22.202Z] INFO on 222 |     at loader (/home/krot47/repos/logger/node_modules/babel-register/lib/node.js:144:5)
[2018-12-25T13:16:22.202Z] INFO on 222 |     at Object.require.extensions.(anonymous function) [as .js] (/home/krot47/repos/logger/node_modules/babel-register/lib/node.js:154:7)
[2018-12-25T13:16:22.202Z] INFO on 222 |     at Module.load (module.js:554:32)
[2018-12-25T13:16:22.202Z] INFO on 222 |     at tryModuleLoad (module.js:497:12)
[2018-12-25T13:16:22.202Z] INFO on 222 |     at Function.Module._load (module.js:489:3)
[2018-12-25T13:16:22.202Z] INFO on 222 |     at Function.Module.runMain (module.js:676:10)
[2018-12-25T13:16:22.202Z] INFO on 222 |     at Object.<anonymous> (/home/krot47/repos/logger/node_modules/babel-cli/lib/_babel-node.js:154:22) } } } } } }
[2018-12-25T13:16:22.203Z] INFO on 222 | [ 1, [ 1, [ 1, [ 1, [ 1, [ 1, [Array] ] ] ] ] ] ]
[2018-12-25T13:16:22.203Z] INFO on 222 | { a: { a: { a: { a: { a: /a/ } } } } }
[2018-12-25T13:16:22.203Z] INFO on 222 | { a: { a: { a: { a: { a: Symbol(asd) } } } } }
[2018-12-25T13:16:22.204Z] INFO on 222 | { a: { a: { a: { a: { a: Promise { 'asd' } } } } } }
