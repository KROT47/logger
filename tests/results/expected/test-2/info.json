{"ts":"2018-12-25T13:27:42.446Z","level":"info","hostname":"222","msgs":[{"x":1,"a":{"x":1,"a":{"x":1,"a":{"x":1,"a":{"x":1,"a":{"x":1,"a":"[Object]"}}}}}}]}
{"ts":"2018-12-25T13:27:42.446Z","level":"info","hostname":"222","msgs":[{"a":{"a":{"a":{"a":{"a":{"a":"[Object]"}}}}}}]}
{"ts":"2018-12-25T13:27:42.446Z","level":"info","hostname":"222","msgs":[{"a":{"a":{"a":{"a":{"a":{"a":"[Array]"}}}}}}]}
{"ts":"2018-12-25T13:27:42.447Z","level":"info","hostname":"222","msgs":[{"a":{"a":{"a":{"a":{"a":{"a":"[Function]"}}}}}}]}
{"ts":"2018-12-25T13:27:42.447Z","level":"info","hostname":"222","msgs":[{"a":{"a":{"a":{"a":{"a":{"a":"[NaN]"}}}}}}]}
{"ts":"2018-12-25T13:27:42.447Z","level":"info","hostname":"222","msgs":[{"a":{"a":{"a":{"a":{"a":{"a":"[Undefined]"}}}}}}]}
{"ts":"2018-12-25T13:27:42.447Z","level":"info","hostname":"222","msgs":[{"a":{"a":{"a":{"a":{"a":{"a":null}}}}}}]}
{"ts":"2018-12-25T13:27:42.447Z","level":"info","hostname":"222","msgs":[{"a":{"a":{"a":{"a":{"a":{"a":1}}}}}}]}
{"ts":"2018-12-25T13:27:42.448Z","level":"info","hostname":"222","msgs":[{"a":{"a":{"a":{"a":{"a":{"a":"1"}}}}}}]}
{"ts":"2018-12-25T13:27:42.448Z","level":"info","hostname":"222","msgs":[{"a":{"a":{"a":{"a":{"a":{"a":"2018-12-25T13:27:42.448Z"}}}}}}]}
{"ts":"2018-12-25T13:27:42.451Z","level":"info","hostname":"222","msgs":[{"a":{"a":{"a":{"a":{"a":{"a":{"message":"","stack":"Error\n    at startTest (/home/krot47/repos/logger/tests/tests/test-2.js:46:48)\n    at Object.<anonymous> (/home/krot47/repos/logger/tests/test.js:47:5)\n    at Module._compile (module.js:635:30)\n    at loader (/home/krot47/repos/logger/node_modules/babel-register/lib/node.js:144:5)\n    at Object.require.extensions.(anonymous function) [as .js] (/home/krot47/repos/logger/node_modules/babel-register/lib/node.js:154:7)\n    at Module.load (module.js:554:32)\n    at tryModuleLoad (module.js:497:12)\n    at Function.Module._load (module.js:489:3)\n    at Function.Module.runMain (module.js:676:10)\n    at Object.<anonymous> (/home/krot47/repos/logger/node_modules/babel-cli/lib/_babel-node.js:154:22)"}}}}}}}]}
{"ts":"2018-12-25T13:27:42.452Z","level":"info","hostname":"222","msgs":[[1,[1,[1,[1,[1,[1,"[Array]"]]]]]]]}
{"ts":"2018-12-25T13:27:42.452Z","level":"info","hostname":"222","msgs":[{"a":{"a":{"a":{"a":{"a":"[/a/]"}}}}}]}
{"ts":"2018-12-25T13:27:42.452Z","level":"info","hostname":"222","msgs":[{"a":{"a":{"a":{"a":{"a":"[Symbol(asd)]"}}}}}]}
{"ts":"2018-12-25T13:27:42.453Z","level":"info","hostname":"222","msgs":[{"a":{"a":{"a":{"a":{"a":"[Promise { 'asd' }]"}}}}}]}
